tests/TestRunner/app/ApiTests.js

```diff
@@ -201,7 +201,7 @@ describe(module.id, function () {
         expect(TNSApi.alloc().init().strokeColor).toBeNull();
     });
 
-    if (TNSIsConfigurationDebug) {
+    if (false && TNSIsConfigurationDebug) {
```

Comment:
The `false &&` part was to disable these lengthy tests while developing, we should remove it.